sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -128,62 +128,108 @@ SHOW TABLE EXTENDED
 
 
 -- !query 13
-SHOW TABLE EXTENDED LIKE 'show_t1' PARTITION(c='Us')
+SHOW TABLE EXTENDED LIKE 'show_t1' PARTITION(c='Us', d=1)
 -- !query 13 schema
-struct<>
+struct<database:string,tableName:string,isTemporary:boolean,information:string>
 -- !query 13 output
-org.apache.spark.sql.catalyst.parser.ParseException
-
-Operation not allowed: SHOW TABLE EXTENDED ... PARTITION(line 1, pos 0)
-
-== SQL ==
-SHOW TABLE EXTENDED LIKE 'show_t1' PARTITION(c='Us')
-^^^
+show_t1
```

Comment:
The `SQLQueryTestSuite` runs the sql and convert the result as a hive compatible sequence of strings. It handles the `ShowTablesCommand` this way:
```
    // SHOW TABLES in Hive only output table names, while ours outputs database, table name, isTemp.
    case command: ExecutedCommandExec if command.cmd.isInstanceOf[ShowTablesCommand] =>
      command.executeCollect().map(_.getString(1))
```
We may either change this case, or we create a new command `ShowTableCommand` to handle the statement `SHOW TABLE EXTENDED LIKE ... [PARTITION]`.